sdk_test.go

```diff
@@ -0,0 +1,17 @@
+package sdk
+
+import (
+	"testing"
+
+	"github.com/ethereum/go-ethereum/rpc"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestNewSDK(t *testing.T) {
+	rpcClient, err := rpc.Dial("http://localhost:8545")
```

Comment:
As you mentioned we need to setup a mock for the rpcClient. 

At status-go we're using [https://github.com/golang/mock](mockgen) to generate mocks based on an interface, in our case should be easy to set it up, as we already have the RPCClient interface created, have a look and ping me if you need help with it :)